qemu/tests/cfg/multi_vms_with_stress.cfg

```diff
@@ -15,9 +15,12 @@
     pre_command = "mkdir -p ${stress_inst_dir}"
     post_command = "killall -9 stress; rm -rf ${stress_inst_dir}"
     ovmf:
-        smp = 1
-        vcpu_maxcpus = 1
-        vcpu_sockets = 1
-        vcpu_dies = 1
-        vcpu_cores = 1
-        vcpu_threads = 1
+        vms = "avocado-vt-vm1 vm2"
+    variants:
+        - @default:
```

Comment:
It looks to me here that the 'default' is used for keeping the original testing logic beore qemu 8.2.0-7, the question is we made some modification above(line 18-23), count of vms is changed from 5 to 2(ovmf), is that expected?

Reply:
Yes, this is also the result of discussion with @qingwangrh . He suggested that I change it like this.

Reply:
@qingwangrh hello Could you add your comment?